django_admin_search/templates/admin/custom_search_form.html

```diff
@@ -1,159 +1,87 @@
-            border: 1px solid #ddd!important;
-            color: #454545!important;
-        }
-
-        .search_button {
-            padding-top: 10px;
-            margin-bottom: 20px;
-        }
-
-        .search_button > a{
-            background-image: none!important;
-            padding: 9px 20px 9px 20px !important;
-            border-radius: 3px!important;
-            color: #454545!important;
-            font-size: 16px!important;
-            text-transform: none!important;
-            letter-spacing: normal!important;
-            background-color: #fff!important;
-            border: 1px solid #ddd!important;
-            background-clip: padding-box;
-        }
-    </style>
-{% endblock %}
+<!-- Stylesheet connection -->
+{% block stylesheet %} {% load static %}
+<link rel="stylesheet" href="{% static 'css/styles.css' %}"> {% endblock %} {% block modal %}
 
-{% block modal %}
+<form id="advanced_admin_search" class="modal" style="display: none;">
 
-    <form id="advanced_admin_search" class="modal" style="display: none;">
-        
-        <div class="advanced_search_form_body">
-            {% block form_fields %}
-                {% include "admin/custom_fieldset.html" %}
-            {% endblock %}
-        </div>
+    <div class="advanced_search_form_body">
+        {% block form_fields %} {% include "admin/custom_fieldset.html" %} {% endblock %}
+    </div>
 
-        {% if show_result_count %}
-            <span class="small quiet">{% blocktrans count counter=cl.result_count %}{{ counter }} result{% plural %}{{ counter }} 
+    {% if show_result_count %}
+    <span class="small quiet">{% blocktrans count counter=cl.result_count %}{{ counter }} result{% plural %}{{ counter }} 
                                       results{% endblocktrans %} (<a href="?{% if cl.is_popup %}_popup=1{% endif %}">
                                       {% if cl.show_full_result_count %}{% blocktrans with full_result_count=cl.full_result_count %}
                                       {{ full_result_count }} total{% endblocktrans %}{% else %}{% trans "Show all" %}{% endif %}</a>)
-            </span>
-        {% endif %}
-        {% for pair in cl.params.items %}
-            {% if pair.0 != search_var %}<input type="hidden" name="{{ pair.0 }}" value="{{ pair.1 }}"/>{% endif %}
-        {% endfor %}
-        
-        {% block modal_search_button %}
-        <button id="submit_form" type="submit" class="form_submit_button">
+            </span> {% endif %} {% for pair in cl.params.items %} {% if pair.0 != search_var %}<input type="hidden" name="{{ pair.0 }}" value="{{ pair.1 }}" />{% endif %} {% endfor %} {% block modal_search_button %}
+    <button id="submit_form" type="submit" class="form_submit_button">
             {% block modal_search_button_text %}
                 {% trans 'filter'|title %}
             {% endblock %}
-        </button>
-        {% endblock %}
+        </button> {% endblock %}
```

Comment:
In this case endblock must be on the next line

because group all this code block is more visual and easy to debug